cobl2/datatables.py

```diff
@@ -481,7 +503,6 @@ def includeme(config):
     # "manually", i.e. in cobl2.main.
     #
     config.register_datatable('values', Forms)
-    config.register_datatable('cognatesets', CognateClasses)
```

Comment:
If the table isn't registered anymore, shouldn't the code be removed?

Reply:
it is already registered in cobl2.main as the note suggest